src/main/java/com/banktaglayouts/BankTagLayoutsConfig.java

```diff
@@ -77,32 +77,40 @@ default boolean shiftModifierForExtraBankItemOptions() {
         return false;
     }
 
-	@ConfigItem(
-		keyName = "autoLayoutDuplicatesEnabled",
-		name = "Auto-layout: Create duplicates",
-		description = "Whether or not to create duplicates when there are multiple of the same item when using auto-layout.",
-		position = 8
-	)
-	default boolean autoLayoutDuplicatesEnabled() {
-		return true;
-	}
+    @ConfigItem(
+            keyName = "autoLayoutAlternateStyle",
+            name = "Auto-layout: Pretty Sort",
+            description = "Organizes the auto layout to look like an inventory and equipment grid.",
+            position = 8
+    )
+    default boolean autoLayoutAlternateStyle() { return true; }
```

Comment:
I would prefer this be an enum so that it becomes a dropdown menu in the runelite config page; currently it sounds like (without reading the tooltip) that it just makes something look prettier instead of being an entirely different style.

It might also be a good idea to take the duplicate settings and put them under their own subsection that's labeled to communicate that they do not affect the pretty sort. I can do that part myself though if you don't want to.

Reply:
Is there an easy way to dynamically hide config options based on which enum is selected? I think it would be best to hide the overflow options when in "Presets" layout mode and show when in "ZigZag" mode

Reply:
No, that is not possible.